pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -44,44 +38,53 @@ buildDotnetModule rec {
 
   preConfigure = ''
     substituteInPlace Directory.Build.props \
-      --replace '</PropertyGroup>' '<ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles></PropertyGroup>'
+      --replace-fail '</PropertyGroup>' '<ErrorOnDuplicatePublishOutputFiles>false</ErrorOnDuplicatePublishOutputFiles></PropertyGroup>'
   '';
 
-  postPatch = ''
-    ln --force --symbolic "${lib.getExe _7zzWithOptionalUnfreeRarSupport}" src/ArchiveManagement/NexusMods.FileExtractor/runtimes/linux-x64/native/7zz
-  '';
+  dotnetBuildFlags = [
+    # Tell the app it is a distro package; affects wording in update prompts
+    "--property:INSTALLATION_METHOD_FLATPAK=1"
+
+    # Don't include upstream's 7zz binary; we use the nixpkgs version
+    "--property:NEXUSMODS_APP_USE_SYSTEM_EXTRACTOR=1"
+  ];
 
   makeWrapperArgs = [
-    "--prefix PATH : ${lib.makeBinPath [desktop-file-utils]}"
+    "--prefix PATH : ${lib.makeBinPath [ desktop-file-utils ]}"
```

Comment:
Would it be enough to just list `desktop-file-utils` in `runtimeDeps`, instead of explicitly adding them to the PATH here?

Runtime dpes all end up on the PATH anyway, right?

Reply:
> Would it be enough to just list `desktop-file-utils` in `runtimeDeps`, instead of explicitly adding them to the PATH here?
> 
> Runtime dpes all end up on the PATH anyway, right?

I think so? I'll give it a try.

Reply:
Did this not work in the end?